### PR TITLE
Httplib2 recipe

### PR DESCRIPTION
**Pull request by npanpaliya**

### CODE REVIEW

**Comment by wdirons** [comment]
@npanpaliya , to get past "the input device is not a TTY" error in my build I had to make this change:  https://github.com/IBM/powerai/pull/190/commits/6fb14550be3d490d3ea6f836f115b2edff1e8747  (which I got from https://stackoverflow.com/a/48230089/5879698)

**Comment by npanpaliya** [comment]
Thanks Billy!

File: conda-recipes/httplib2-feedstock/LICENSE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+BSD 3-clause license
+Copyright (c) 2015-2019, conda-forge, IBM
```
Comment (deepali-c) [inline conda-recipes/httplib2-feedstock/LICENSE]: Do we need to update this to 2020?

File: conda-recipes/httplib2-feedstock/ci_support/build_steps.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# (C) Copyright IBM Corp. 2018, 2019. All Rights Reserved.
```
Comment (deepali-c) [inline conda-recipes/httplib2-feedstock/ci_support/build_steps.sh]: Same as previous, copyright needs to be updated to 2020.

File: conda-recipes/httplib2-feedstock/ci_support/run_docker_build.sh [hunk 3]
```diff
@@ -0,0 +1,60 @@
+# (C) Copyright IBM Corp. 2018, 2019. All Rights Reserved.
```
Comment (deepali-c) [inline conda-recipes/httplib2-feedstock/ci_support/run_docker_build.sh]: Copyright update to 2020.

File: conda-recipes/httplib2-feedstock/ci_support/run_docker_build.sh [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+if hash docker-machine 2> /dev/null && docker-machine active > /dev/null; then
+    HOST_USER_ID=$(docker-machine ssh $(docker-machine active) id -u)
+fi
+
+ARTIFACTS="$FEEDSTOCK_ROOT/build_artifacts"
+mkdir -p "$ARTIFACTS"
+DONE_CANARY="$ARTIFACTS/conda-forge-build-done-${CONFIG}"
+rm -f "$DONE_CANARY"
+# Enable running in interactive mode attached to a tty
+test -t 1 && USE_TTY="-t"
```
Comment (deepali-c) [inline conda-recipes/httplib2-feedstock/ci_support/run_docker_build.sh]: Probably we can add a comment for this change here.

File: conda-recipes/httplib2-feedstock/LICENSE (hunk 1 above)
Comment (npanpaliya) [reply conda-recipes/httplib2-feedstock/LICENSE]: No, this comes from httplib2. I don't think we should update this.

Comment (deepali-c) [reply conda-recipes/httplib2-feedstock/LICENSE]: ok

**Pull request merged**